config/brokers/channel-broker/deployments/controller.yaml

```diff
@@ -1,75 +0,0 @@
-# Copyright 2018 The Knative Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: broker-controller
-  namespace: knative-eventing
-  labels:
-    eventing.knative.dev/release: devel
-spec:
-  replicas: 1
```

Comment:
should we keep this with `replicas: 0` ?   

We have done this before with other "deprecated" PRs.

Thoughts?

Reply:
So, if we do this, we can't remove this?
https://github.com/vaikas/eventing/blob/remove-channel-based-broker/hack/release.sh#L24

Just have the deployment there, set to 0 replicas?

Reply:
so remove this file and move to mt-channel-broker? Or? :) Confused :)

Reply:
I think we want to delete the file outright and add a followup to do a cluster update for deleting a `deployment` in the `knative-eventing` namespace called `broker-controller`